package.json

```diff
@@ -33,18 +33,18 @@
   },
   "dependencies": {
     "chalk": "^2.4.2",
-    "cli-table3": "^0.5.1",
+    "cli-table3": "^0.6.0",
     "express": "^4.17.1",
     "express-redirect": "^1.2.2",
     "http-proxy": "^1.18.1",
     "lodash.defaults": "^4.2.0",
-    "yargs": "^14.2.0"
+    "yargs": "^17.3.0"
   },
   "devDependencies": {
-    "eslint": "^6.8.0",
+    "eslint": "7.32.0",
```

Comment:
Is there a reason for having non-caret version here? If possible, use "^7.32.0" (use `yarn upgrade eslint@^7.32.0`)